src/web/src/pages/myfeeds.tsx

```diff
@@ -2,13 +2,14 @@ import SEO from '../components/SEO';
 import BackToTopButton from '../components/BackToTopButton';
 import useAuth from '../hooks/use-auth';
 import MyFeeds from '../components/MyFeeds';
+import { telescopeUrl } from '../config';
 
 const MyFeedsPage = () => {
   const { login, user } = useAuth();
 
   // Redirect the user to login if they aren't authenticated already
   if (!user) {
-    return login('/myfeeds');
+    return login(`${telescopeUrl}/myfeeds`);
```

Comment:
I wonder if we should change the way that `login` works so that you can pass us a relative path, and we'll do the `${telescopeUrl}/` prefix for you.  That way people don't have to pull in `telescopeUrl` from config in order to accomplish this.

If so, `login` would need to be updated to something like this:

```tsx
    const url = new URL(returnTo || '', telescopeUrl)
    window.location.href = `${loginUrl}?redirect_uri=${url.href}&state=${loginState}`;
```

Reply:
Will we need to call `encodeURIComponent` on `url` ?